sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_conversation_app_async.py

```diff
@@ -30,47 +32,65 @@ async def sample_analyze_conversation_app_async():
     from azure.core.credentials import AzureKeyCredential
 
     from azure.ai.language.conversations.aio import ConversationAnalysisClient
-    from azure.ai.language.conversations.models import ConversationAnalysisOptions
+    from azure.ai.language.conversations.models import DateTimeResolution
 
     # get secrets
-    conv_endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    conv_project = os.environ["AZURE_CONVERSATIONS_PROJECT"]
-
-    # prepare data
-    query = "One california maki please."
-    input = ConversationAnalysisOptions(
-        query=query
-    )
-
-    # analyze query
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_CONVERSATIONS_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_CONVERSATIONS_DEPLOYMENT_NAME"]
+
+    # analyze quey
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=conv_project,
-            deployment_name='production'
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = await client.analyze_conversation(
+            body={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
+                        "modality": "text",
+                        "language": "en",
+                        "text": query
+                    },
+                    "isLoggingEnabled": False
+                },
+                "parameters": {
+                    "projectName": project_name,
+                    "deploymentName": deployment_name,
+                    "verbose": True
+                }
+            }
         )
 
         # view result
-        print("query: {}".format(result.query))
-        print("project kind: {}\n".format(result.prediction.project_kind))
-
-        print("view top intent:")
-        print("\ttop intent: {}".format(result.prediction.top_intent))
-        print("\tcategory: {}".format(result.prediction.intents[0].category))
-        print("\tconfidence score: {}\n".format(result.prediction.intents[0].confidence_score))
-
-        print("view entities:")
-        for entity in result.prediction.entities:
-            print("\tcategory: {}".format(entity.category))
-            print("\ttext: {}".format(entity.text))
-            print("\tconfidence score: {}".format(entity.confidence_score))
+        print("query: {}".format(result.results.query))
+        print("project kind: {}\n".format(result.results.prediction.project_kind))
```

Comment:
hmm I wonder if we can use overloads to avoid the wrapper object around the result here....

Reply:
can you please elaborate? i don't quite understand

Reply:
Ignore this for now - I need to do some experiments on the API to see what would/would not be possible... either way - not important for this preview.
Essentially I want to see if there's a way we can avoid the `result.results` repetition - assuming we can deduce the output type from the input type..... but need to test it.